code/sys/sys_win32.c

```diff
@@ -222,6 +225,17 @@ char *Sys_GogPath( void )
 	return gogPath;
 }
 
+/*
+================
+Sys_MicrosoftStorePath
+================
+*/
+char* Sys_MicrosoftStorePath(void)
+{
+	Q_strcat(microsoftStorePath, MAX_OSPATH, "C:\\Program Files\\ModifiableWindowsApps\\Quake 3\\EN");
```

Comment:
Q_strcat() will append path each time Sys_MicrosoftStorePath() is called.

Reply:
I've addressed this in more recent checkins.